modules/mlflow/applications.tf

```
@@ -22,7 +22,8 @@ module "mlflow_mysql" {
   channel         = "8.0/stable"
   # The following config is equivalent to "constraints: mem=2G"
   config = {
-    profile-limit-memory = "2048"
+    profile-limit-memory   = "2048",
+    enable_mlflow_nodeport = var.enable_mlflow_nodeport,
   }
```

This should be part of mlflow_server config, not mysql.

---

Also I think it should look like:

```suggestion
    mlflow_nodeport = var.enable_mlflow_nodeport,
  }
```
according to the charm's config https://github.com/canonical/mlflow-operator/blob/main/config.yaml#L21.

---

> Also I think it should look like:

the config you linked is the nodeport string, what this PR is targeted to is the boolean rather on [L36](https://github.com/canonical/mlflow-operator/blob/main/config.yaml#L36)

Do you think we should also add a config for the nodeport string?

---

You 're right about my confusion, I was looking at the wrong config option. Sure, let's add the port configuration, it makes sense.

---

updated